Tests/SwiftSyntaxTest/SyntaxTests.swift

```diff
@@ -23,7 +23,9 @@ public class SyntaxAPITestCase: XCTestCase {
     XCTAssertEqual("\(funcKW.previousToken!)", "{ ")
 
     var toks = [TokenSyntax]()
-    funcKW.parent!.forEachToken { toks.append($0) }
+    for token in funcKW.parent!.tokens {
```

Comment:
Nitpick (not essential, but a good outcome of switching to sequences):

```swift
let toks = Array(funkKW.parent!.tokens)
```

Reply:
Nice! Done.